imports/plugins/core/dashboard/client/components/SettingsList.js

```diff
@@ -27,34 +47,37 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function SettingsList() {
   const classes = useStyles();
-  const history = useHistory();
   const settingsRoutes = useOperatorRoutes({ groups: ["settings"] });
   let settingsList = [];
 
   if (Array.isArray(settingsRoutes)) {
     settingsList = settingsRoutes.map((setting) => (
-      <ListItem
-        key={`listItem-${setting.path}`}
-        component="nav"
-        ContainerProps={{
-          className: classes.listItemContainer
-        }}
-        ContainerComponent={({ children, ...props }) => (
-          <li {...props}>
-            {i18next.t(setting.sidebarI18nLabel) }
-            {children}
-          </li>
-        )}
-        className={clsx({
-          [classes.listItem]: true
-        })}
-        button
-        onClick={() => history.push(setting.path)}
+      <NavLink
+        activeClassName={activeClassName}
+        className={clsx({ [classes.link]: true, [classes.linkBackground]: true })}
+        to={setting.path}
+        key={setting.path}
       >
-        <ListItemText
-          primary={i18next.t(setting.sidebarI18nLabel)}
-        />
-      </ListItem>
+        <ListItem
+          component="nav"
+          ContainerProps={{
+            className: classes.listItemContainer
+          }}
+          button={false}
```

Comment:
I added a ticket in catalyst to revisit the list styles as well because every instance we have of them (main nav, products editor, settings) are all custom jobs, but they should be more reusable.

https://github.com/reactioncommerce/catalyst/issues/155